.circleci/check_docstring_format.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+match=`git grep -A 1 Parameters | grep "\-\-\-"`
```

Comment:
The proposed `grep` will do the trick if `Parameters` is used instead of [Args:](https://github.com/pytorch/vision/issues/3003#issuecomment-728051622) but it won't capture the other issues listed on the ticket #3003. 

As discussed offline, it would be great to see if any of the popular python doc lints (`darglint`, `pydocstyle` or something else) can help us verify the format of pydocs for future PRs. As you pointed out, lints can enforce very strict rules that are not helpful to our community so perhaps it's worth using only the minimum checks related to missing/incorrect arguments on existing pydocs. 

Let me know if you find any package that checks the above boxes or you thing of an alternative approach.

Reply:
OK, I removed the CI check and I'll investigate how we can enforce 1) consistency between a function's signature and its documented parameters, and 2) the dostring style (google style vs numpy vs ...)

We can probably merge this PR as-is however, since it's a net improvement on its own